cmd/defenseclaw-setup/maintenance_gateway.go

```
@@ -0,0 +1,130 @@
+	run connectorLifecycleRunner,
+) connectorReconciliationRecorder {
+	if len(transaction.PreviousConnectors) == 0 {
+		return connectorReconciliationRecorder{}
+	}
+	maintenance, err := provide()
+	if err != nil {
+		reconciliation := connectorReconciliationRecorder{}
+		for _, connectorName := range transaction.PreviousConnectors {
+			configHome := connectorConfigHome(transaction, connectorName, true)
+			reconciliation.run(transaction.ID, connectorName, configHome, "payload-missing", func() error {
+				return fmt.Errorf("Setup-owned connector maintenance payload is unavailable; cleanup was not attempted: %w", err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)